src/main/resources/api/service_openapi.yaml

```diff
@@ -121,8 +119,7 @@ components:
       properties:
         resourceUid:
           # CloudResourceUid is referred from CRL's cloud-resource-schema.
-          description: The UID of the resource in CRL format
-          $ref: '../../../../build/generated/cloud_resources_uid.yaml#/components/schemas/CloudResourceUid'
+          $ref: '../../../../build/cloud_resources_uid.yaml#/components/schemas/CloudResourceUid'
```

Comment:
putting cloud_resources_uid.yaml directly in the top level of `build` seems odd. Maybe put it in some subdirectory under `build` even if it can't live in the same directory as what's created by swagger code gen.

Reply:
Created another dir for that. Thanks!